OpenSim/Simulation/Manager/Manager.h

```diff
@@ -240,13 +251,15 @@ class OSIMSIMULATION_API Manager
     * finalTime = 10.0;
     * int n = int(round(finalTime/dTime));
     * state.setTime(0.0);
+    * manager.initialize(state);
     * for (int i = 1; i <= n; ++i) {
-    *     manager.integrate(state, i*dTime);
+    *     manager.integrate(i*dTime);
     * }
     * @endcode
     *
     */
-    bool integrate(SimTK::State& s, double finalTime);
+    bool integrate(double finalTime);
```

Comment:
Since we hold a reference (pointer) to the *state* passed in during `initialize`, should we also have an accessor to return the updated *state*? I'm OK with not, since the user passes the state in, but we have to make it more clear that the state is being updated by `integrate` (even though it is not an input to this method).

Reply:
I talked with @aseth1 about this. We thought it could be nice to do the following:

1. Make a copy of the state passed into `initialize()`
2. Change the signature of `integrate()` to return the final state. That is, user code would look like `auto finalState = manager.integrate(tf)`.

This way, it's clear where to get the final state values from. Also, I think users might find it confusing that edits to the state passed into `initialize()` have no effect after calling `initalize()`, but that this same state object gets magically updated after integrating.